webcomponents/tool/src/main/frontend/js/conversations/sakai-topic-summary.js

```diff
@@ -111,8 +111,8 @@ export class SakaiTopicSummary extends SakaiElement {
           ${this.topic.numberOfUnreadPosts > 0 ? html`
             <sakai-icon type="circle"
                 size="small"
-                aria-label="${this.i18n.read_tooltip}"
-                title="${this.i18n.read_tooltip}">
```

Comment:
Is this property used elsewhere? If it isn't, then it should be removed from all the files.

Reply:
Maybe not. I'll check.

Reply:
removed. Good spot